renderer/src/components/UpdateBanner.tsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from 'react'
+import { ReactComponent as Warning } from '../assets/img/error.svg'
+import { openReleaseNotes, restartToUpdate } from '../lib/station-config'
+
+const UpdateBanner = () => {
+  const [isUpdateAvailable, setIsUpdateAvailable] = useState<boolean>(false)
+
+  useEffect(() => {
+    window.electron.getUpdaterStatus()
+      .then(({ updateAvailable }) => setIsUpdateAvailable(updateAvailable))
```

Comment:
can we please again use `async/await`?

Reply:
[ba42f5d](https://github.com/filecoin-station/filecoin-station/pull/199/commits/ba42f5d4b1b6fcb8c767d54a77bf71842a7eef8e)